pymc/tests/test_sampling.py

```diff
@@ -1128,6 +1129,128 @@ def test_logging_sampled_basic_rvs_posterior_deterministic(self, caplog):
         assert caplog.record_tuples == [("pymc", logging.INFO, "Sampling: [y, z]")]
         caplog.clear()
 
+    @pytest.fixture(scope="class")
+    def mock_multitrace(self):
+        rng = np.random.default_rng(seed=42)
+        data = rng.normal(loc=1, scale=0.2, size=(10, 3))
+        with pm.Model() as model:
+            model.add_coord("name", ["A", "B", "C"], mutable=True)
+            model.add_coord("obs", list(range(10, 20)), mutable=True)
+            offsets = pm.MutableData("offsets", rng.normal(0, 1, size=(10,)))
+            a = pm.Normal("a", mu=0, sigma=1, dims=["name"])
+            b = pm.Normal("b", mu=offsets, sigma=1)
+            mu = pm.Deterministic("mu", a + b[..., None], dims=["obs", "name"])
+            sigma = pm.HalfNormal("sigma", sigma=1, dims=["name"])
+
+            data = pm.MutableData(
+                "y_obs",
+                data,
+                dims=["obs", "name"],
+            )
+            pm.Normal("y", mu=mu, sigma=sigma, observed=data, dims=["obs", "name"])
+            trace = pm.sample(
+                draws=10,
+                tune=10,
+                chains=2,
+                progressbar=False,
+                compute_convergence_checks=False,
+                return_inferencedata=False,
+            )
+        return trace
+
+    @pytest.fixture(scope="class", params=["MultiTrace", "InferenceData", "Dataset"])
+    def mock_sample_results(self, request, mock_multitrace):
+        kind = request.param
+        trace = mock_multitrace
+        # We rebuild the class to ensure that all dimensions, data and coords start out
+        # the same across params values
+        rng = np.random.default_rng(seed=42)
+        data = rng.normal(loc=1, scale=0.2, size=(10, 3))
+        with pm.Model() as model:
+            model.add_coord("name", ["A", "B", "C"], mutable=True)
+            model.add_coord("obs", list(range(10, 20)), mutable=True)
+            offsets = pm.MutableData("offsets", rng.normal(0, 1, size=(10,)))
+            a = pm.Normal("a", mu=0, sigma=1, dims=["name"])
+            b = pm.Normal("b", mu=offsets, sigma=1)
+            mu = pm.Deterministic("mu", a + b[..., None], dims=["obs", "name"])
+            sigma = pm.HalfNormal("sigma", sigma=1, dims=["name"])
+
+            data = pm.MutableData(
+                "y_obs",
+                data,
+                dims=["obs", "name"],
+            )
+            pm.Normal("y", mu=mu, sigma=sigma, observed=data, dims=["obs", "name"])
+        if kind == "MultiTrace":
+            return kind, trace, model
+        else:
+            idata = pm.to_inference_data(
+                trace,
+                save_warmup=False,
+                model=model,
+                log_likelihood=False,
+            )
+            if kind == "Dataset":
+                return kind, idata.posterior, model
+            else:
+                return kind, idata, model
+
+    def test_mutable_coords_and_data(self, mock_sample_results, caplog):
```

Comment:
It's a bit of a mouthful, but we are only checking the logging so it's more appropriate. If you remove the test mentioned above, I think you can just borrow its name instead: `test_logging_sampled_basic_rvs_posterior_mutable`

```suggestion
    def test_logging_sampled_basic_rvs_posterior_mutable_coords_and_data(self, mock_sample_results, caplog):
```

Reply:
I'll do this